tasks/configure.yml

```diff
@@ -17,8 +25,8 @@
   service: name={{ mongodb_daemon_name }} state=started enabled=yes
 
 - name: wait MongoDB port is listening
-  wait_for: host="{{ mongodb_conf_bind_ip }}"port="{{ mongodb_conf_port }}" delay=10 timeout=60 state=started
-  when: "'systemd' in systemd.stdout"
+  wait_for: host="{{ mongodb_listen_iface }}" port="{{ mongodb_conf_port }}" delay=5 timeout=60 state=started
+  #when: "'systemd' in systemd.stdout" #TODO Why only with systemd?
```

Comment:
I don't know, another user have problem with this task, because mongo service is not started.